src/main/java/com/datalogics/pdf/hsm/samples/HsmManager.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2016 Datalogics Inc.
+ */
+
+package com.datalogics.pdf.hsm.samples;
```

Comment:
From yesterday, our agreed package name for this interface is `com.datalogics.pdf.security`. (Note that this package does not currently exist in any repo.)